examples/nidcpower_source_dc_voltage/README.md

```diff
@@ -43,18 +42,18 @@ created in NI MAX. To automatically simulate an instrument without using NI MAX,
 follow the steps below:
 - Create a `.env` file in the measurement service's directory or one of its
   parent directories (such as the root of your Git repository or
-  `C:\ProgramData\National Instruments\MeasurementLink\Services` for statically
+  `C:\ProgramData\National Instruments\Plug-Ins\Measurements` for statically
   registered measurement services).
 - Add the following options to the `.env` file to enable simulation via the
   driver's option string:
 
   ```
-  MEASUREMENTLINK_NIDCPOWER_SIMULATE=1 
+  MEASUREMENTLINK_NIDCPOWER_SIMULATE=1
```

Comment:
@dixonjoel @DavidCurtiss Outside the scope of this PR: will these also be renamed?

Reply:
I think we should at some point. I'm not too concerned if it makes this release or not. We should probably have an issue for this to remind us at least. I'll file one.

Reply:
https://github.com/ni/measurement-plugin-python/issues/757